helm/hpcc/templates/thor.yaml

```diff
@@ -42,6 +42,7 @@ spec:
                 {{ printf "\"--name=%s\"" .name }}
               ]
 {{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 8 }}
+{{- include "hpcc.addResources" (dict "root" $ "me" .) | indent 8 }}
```

Comment:
It isn't very clear that resources defined for thor will actually apply to the eclagent, and that thor resources need to be configured with childContainerResources

Reply:
The resources define the limits for pods created directly by helm.  The values are copied over automatically by helm and used to set the limits for pods that it starts.

ChildContainerResources are for any pods created by one of our processes.  Any pods created using *-jobspec.yaml use the ChildContainerResources.

Reply:
perhaps it should be flattened, so that user can supply an optional

- jobAgentContainerResources: 
- thorAgentContainerResources:
- thorWorkerContainerResources:
- thorManagerContinerResources:

I suspect the 1st 2, which are v. lightweight would generally not be used.
Each of these can be copied (if present) into the appropriate generates "resources:" place in the generated yaml, along with any workunit overrides if present.

Reply:
@jakesmith Would these objects be under the Global section?  If that is the case, would overrides be supported under individual components section?

Reply:
I had only been thinking of them as nodes under the thor instance name (e.g. under thor:).
But yes, I think as with defaults per component type (as per Gavin's point: https://github.com/hpcc-systems/HPCC-Platform/pull/14108#discussion_r487836236), you'd want a way to define global defaults per these component subtype too.

Reply:
That's not a problem.  I have implemented something similar already, although the names are a little different.